docs/Linuxbrew.md

```diff
@@ -0,0 +1,99 @@
+![Linuxbrew logo](https://linuxbrew.sh/images/linuxbrew-256x256.png)
+
+# Linuxbrew
+
+The Homebrew package manager may be used on Linux and Windows 10+, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about), as well as macOS. Homebrew is referred to as Linuxbrew when running on Linux or Windows. It can be installed in your home directory and does not require root access. Recent versions of *glibc* and *gcc* may be installed in your home directory on an older distribution of Linux. A Linuxbrew installation does not use any libraries provided by your host system, except *glibc* and *gcc* if they are sufficiently current.
+
+[Features](#features), [dependencies](#dependencies) and [installation instructions](#install) are described below. Terminology (e.g. the difference between a Cellar, Tap, Cask and so forth) is [explained in the documentation](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology).
+
+# Install
+
+The installation script installs Linuxbrew to `/home/linuxbrew/.linuxbrew` if possible and in your home directory at `~/.linuxbrew` otherwise. Using `/home/linuxbrew/.linuxbrew` allows the use of more binary packages, called bottles, than installing in your personal home directory.
```

Comment:
`, called bottles,` ~> `("bottles")`  or `,aka "bottles",` or `("bottles" in Homebrew terminology)`
`than` ~> `rather than`

Reply:
I generally prefer parenthetical commas over parentheses in my writing. I don't feel strongly about this one. Perhaps

… binary packages (bottles) than installing …

Reply:
```
$ rg 'binary.*bottle|bottle.*binary'
diagnostic.rb:          You can install Homebrew anywhere you want but some bottles (binary packages)
brew.sh:you being able to use all the binary packages (bottles) Homebrew provides. We
build_options.rb:  # True if a {Formula} is being built as a bottle (i.e. binary package).
formula.rb:  # When installing a bottle (binary package) from a local path this will be
cmd/vendor-install.sh:both Bintray (used for Homebrew bottles/binary packages) and GitHub
dev-cmd/bottle.rb:#:    Generate a bottle (binary package) from a formula that was installed with
dev-cmd/bottle.rb:        Generate a bottle (binary package) from a formula that was installed with
extend/os/linux/diagnostic.rb:          You will be unable to use binary packages (bottles).
```

Reply:
Looks like parenthetical style for defining bottles is more common in the Homebrew codebase.

Reply:
> if possible

when would it not be possible?

agreed with matching existing style

Reply:
> The installation script installs Linuxbrew to `/home/linuxbrew/.linuxbrew` if possible

> when would it not be possible?

When the user does not have `sudo` permissions on their system.